### ISSUE TITLE
Using a parameter as Y axis in scatter plot

### ISSUE THREAD

**Issue by lemairecarl** [comet-ml/issue-tracking#449]
## Describe the Bug
When trying to use a parameter as Y axis in scatter plot, it wrongly says "No data available for this chart"

## Expected behavior
I want to do a scatter plot with the HPO trial number as X, and weight_decay as Y. Both are parameters, not metrics. This would be useful in an Hyperparameter Optimization context.

## Where is the issue?
- [ ] Comet Python SDK
- [x] Comet UI
- [ ] Third Party Integrations (Huggingface, TensorboardX, Pytorch Lighting etc)

## To Reproduce
1. Add a scatter plot
2. Select params as X and Y axis
3. It says "No data available for this chart"

## Link to Comet Project/Experiment
If applicable, please provide a link to your Comet Project or Experiment. 
https://www.comet.ml/lemairecarl/microscopie/view/lzy4th53wbc8iEZCjpV3eh2kj/panels

**Comment by DN6** [comment]
@lemairecarl Would it be possible to make your project public so that I can try reproducing this issue on my end?

**Comment by lemairecarl** [comment]
Sorry, it's now public.

For example, you can try creating a scatter plot using X="optuna_trial" and Y="log_wd".

Thanks.

**Comment by DN6** [comment]
Hi @lemairecarl. I was able to reproduce your issue. Indeed, this looks like a bug. Thank you for catching it. I will notify the engineering team and update you here when the issue is resolved.
